app/src/main/java/com/example/triptracker/view/map/RecordScreen.kt

```diff
@@ -423,10 +422,8 @@ fun Map(
                           colors =
                               OutlinedTextFieldDefaults.colors(
                                   unfocusedTextColor = md_theme_light_onPrimary,
-                                  unfocusedBorderColor =
-                                      if (isTitleEmpty) md_theme_light_error else md_theme_grey,
-                                  unfocusedLabelColor =
-                                      if (isTitleEmpty) md_theme_light_error else md_theme_grey,
+                                  unfocusedBorderColor = getBorderColor(isTitleEmpty),
+                                  unfocusedLabelColor = getBorderColor(isTitleEmpty),
```

Comment:
Clever choice that you remove these nested conditions check to put it in another function for complexity ! Greatly appreciated.